model/task/task.go

```diff
@@ -2741,6 +2742,37 @@ func GetTasksByVersion(versionID string, sortBy []TasksSortOrder, statuses []str
 					tempParentKey: []interface{}{},
 				},
 			},
+			// get any annotation that has at least one issue
```

Comment:
Do we need annotation information everywhere we use this query? If we don't we can pass in a boolean and only add this lookup to the pipeline if we need to.

Reply:
It looks like it is only used by the resolver, so I believe yes.